src/app/register/page.tsx

```diff
@@ -11,6 +12,11 @@ const Register = () => {
   const defaultStyle =
     "no-underline	text-cream border border-transparent transition-all duration-200 p-[3px] rounded";
 
+    const [name,setName] : string = useState("");
```

Comment:
- I suggest you to use `Zod` for form validation and `react-hook-form` for form submission